File: jokenpo.py
from random import *


def choose_computer():
    computer = randint(0, 2)
    return computer

def choose_player():
    player = int(input('''
        Vamos jogar JOKENPÔ!
            0 - Pedra
            1 - Papel
            2 - Tesoura
            :'''))
    return player


def winner():
    list = ['Pedra', 'Papel', 'Tesoura']

    player_result = choose_player()
    computer_result = choose_computer()

    if computer_result == player_result:
        print(f'Player escolheu: {list[player_result]}')
        print(f'Computer escolher: {list[computer_result]}')
        print('Empate')
    elif (computer_result == 0 and player_result == 1) \
            or (computer_result == 1 and player_result == 2) \
            or (computer_result == 2 and player_result == 0):
        print(f'Player escolheu: {list[player_result]}')
        print(f'Computer escolher: {list[computer_result]}')
        print('Você Ganhou!')
    else:
        print(f'Player escolheu: {list[player_result]}')
        print(f'Computer escolher: {list[computer_result]}')
        print('Computer Ganhou')

def main():
    winner()

main()

File: Jokenpo.java
package jokenpo;

import java.util.ArrayList;
import java.util.List;
import java.util.Random;
import java.util.Scanner;

public class Jokenpo {

    public static int choose_computer(){
        
        Random random = new Random();
        int computer = random.nextInt(3);
        
        return computer;   
    } 
    
    public static int choose_player(){
        
        Scanner input = new Scanner(System.in);
        
        System.out.println("Vamos jogar JOKENPÔ!");
        System.out.println("0 - Pedra");
        System.out.println("1 - Papel");
        System.out.println("2 - Tesoura");

        int player = input.nextInt();
        
        return player;
    }
    
    public static void winner(){
        
        List list = new ArrayList();
        list.add("Pedra");
        list.add("Papel");
        list.add("Tesoura");
                
        int player_result = choose_player();
        int computer_result = choose_computer();

        if(computer_result == player_result){
            System.out.println("Player escolheu: "+list.get(player_result));
            System.out.println("Computer escolheu: "+list.get(computer_result));
            System.out.println("Empate");
        }else if((computer_result == 0 && player_result == 1)|| 
                (computer_result == 1 && player_result == 2) ||
                (computer_result == 2 && player_result == 0)){
            System.out.println("Player escolheu: "+list.get(player_result));
            System.out.println("Computer escolheu: "+list.get(computer_result));
            System.out.println("Você Ganhou");
        }else{
            System.out.println("Player escolheu: "+list.get(player_result));
            System.out.println("Computer escolheu: "+list.get(computer_result));
            System.out.println("Computer Ganhou");
        }    
    }

    public static void main(String[] args) {
        winner();
    } 
}
